packages/esm-patient-test-results-app/src/overview/recent-overview.component.tsx

```diff
@@ -2,10 +2,12 @@ import React from "react";
 
 import Button from "carbon-components-react/lib/components/Button";
 import DataTableSkeleton from "carbon-components-react/lib/components/DataTableSkeleton";
+
+import { switchTo, useCurrentPatient } from "@openmrs/esm-framework";
```

Comment:
Would not use `useCurrentPatient`. The props already supply the current patient.